build.sbt

```diff
@@ -81,6 +82,7 @@ lazy val `support-frontend` = (project in file("support-frontend"))
     (testOnly in Test) := ((testOnly in Test) dependsOn testScalastyle).evaluated,
     (testQuick in Test) := ((testQuick in Test) dependsOn testScalastyle).evaluated,
   ).dependsOn(`support-services`, `support-models`, `support-config`, `support-internationalisation`)
+  .aggregate(`support-services`, `support-models`, `support-config`, `support-internationalisation`)
```

Comment:
previously the support-frontend build used to run test in the root, which would run all the tests in all projects.  As part of the snyk monitor changes we removed that extra step as we decided it was slowing things down, and we made sure the support-workers build was also reported to the PR status.
I just realised that in that case the tests for the depended on projects were not being run, as they were only aggregated at the root. https://www.scala-sbt.org/1.x/docs/Multi-Project.html#Aggregation

This is overall a change from before.  Originally if the tests were broken anywhere, support frontend wouldn't build, but support-workers would build unless its own direct tests were broken.
Now, support-workers and support-frontend will go out if their own test and dependent tests are all passing, but if their own tests only are broken, the other would still build successfully.

I don't feel like that's an issue, but there may be other effects I don't know of?